### PR TITLE
Add Docker file sharing implementation to known issues

### PR DESCRIPTION
**Pull request by MuhammadMurtaza**

### CODE REVIEW

**Comment by phenaproxima** [comment]
Looks like there are merge conflicts...?

**Comment by abhisekmazumdar** [comment]
Oh, I noticed that @MuhammadMurtaza has already submitted the PR. That's cool.

I'm sharing the readme changes I was making to include in this PR. Feel free to use it or tweak it.
```
### Error: SQLSTATE[HY000]: General error: 11 database disk image is malformed

If you're using **ddev with Docker Desktop on a Mac**, you might see the following error:

`SQLSTATE[HY000]: General error: 11 database disk image is malformed`

This error usually happens because of the way files are shared between your Mac and Docker, which is set to `VirtioFS`.

To fix this error, you need to change the file sharing method to either `gRPC FUSE` or `osxfs (Legacy)`.

Here's how you can change the file sharing method:

- Open the Docker Desktop settings: [Docker Desktop settings for Mac](https://docs.docker.com/desktop/settings/mac/)
- Look for the "General" tab, and find the option for file sharing implementation.
- Choose either `gRPC FUSE` or `osxfs (Legacy)` from the options available.
- After making the change, click on **Apply & Restart** docker.
```

**Comment by abhisekmazumdar** [comment]
@MuhammadMurtaza Feel free to update your pull request with the suggested text from the referenced comment: https://github.com/phenaproxima/starshot-prototype/pull/66#issuecomment-2107631696

And thank you so much for finding out the root cause. It was really frustrating me that I was not able to try out starshot at my end.

**Comment by phenaproxima** [comment]
Thanks for the fantastic detective work and documentation, folks. This kind of collaboration is what's going to make Starshot awesome!!

File: README.md [hunk 1]
```diff
@@ -98,3 +98,20 @@ If you encounter this, you can restart the server using the following command:
 ```
 composer drupal:run-server
 ```
+
+### Error: SQLSTATE[HY000]: General error: 11 database disk image is malformed
+
```
Comment (phenaproxima) [inline README.md:103]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -98,3 +98,20 @@ If you encounter this, you can restart the server using the following command:
...
+This error usually happens because of the way files are shared between your Mac and Docker, which is set to `VirtioFS`.
+
+To fix this error, you need to change the file sharing method to either `gRPC FUSE` or `osxfs (Legacy)`.
+
+Here's how you can change the file sharing method:
+
+- Open the Docker Desktop settings: [Docker Desktop settings for Mac](https://docs.docker.com/desktop/settings/mac/)
+- Look for the "General" tab, and find the option for file sharing implementation.
+- Choose either `gRPC FUSE` or `osxfs (Legacy)` from the options available.
+- After making the change, click on **Apply & Restart** docker.
```
Comment (phenaproxima) [inline README.md:117]:
```suggestion
- After making the change, click on **Apply & Restart** Docker.
```

**Review by phenaproxima — APPROVED** [review]
Minor things.

File: README.md [hunk 3]
```diff
@@ -98,3 +98,19 @@ If you encounter this, you can restart the server using the following command:
 ```
 composer drupal:run-server
 ```
+
+### Error: SQLSTATE[HY000]: General error: 11 database disk image is malformed
+If you're using **ddev with Docker Desktop on a Mac**, you might see the following error:
```
Comment (phenaproxima) [inline README.md:103]:
```suggestion
If you're using **DDEV with Docker Desktop on a Mac**, you might see the following error:
```

File: README.md [hunk 4]
```diff
@@ -98,3 +98,19 @@ If you encounter this, you can restart the server using the following command:
...
+
+`SQLSTATE[HY000]: General error: 11 database disk image is malformed`
```
Comment (phenaproxima) [inline README.md:105]:
```suggestion
\```
`SQLSTATE[HY000]: General error: 11 database disk image is malformed`
\```
```

**Comment by MuhammadMurtaza** [comment]
Updated with @abhisekmazumdar detailed and concise text. Sorry didn't realize @abhisekmazumdar had made PR before me.

**Pull request merged**